recipes-bsp/u-boot/u-boot-hab.inc

```diff
@@ -5,27 +5,31 @@ DEPENDS += "util-linux-native"
 
 SRC_URI:append = "\
     file://u-boot-hab.cfg \
-    file://create_csf.sh \
     file://create_fuse_cmds.sh \
+"
+
+TDX_IMX6_IMX7_CSF_ARTIFACTS = "\
+    file://imx6_create_csf.sh \
```

Comment:
Similar to Sergio's question. This script is for imx6 and imx7, correct?

Reply:
Correct. I'll make that change as well.

Reply:
Updated as per suggestion.